tests/crypto/secp256r1/testcase.yaml

```
@@ -3,7 +3,7 @@ common:
           CONFIG_CSPRNG_ENABLED and
           CONFIG_HARDWARE_DEVICE_CS_GENERATOR and
           not CONFIG_BUILD_WITH_TFM and
-          not (CONFIG_MBEDTLS and CONFIG_TINYCRYPT)
+          not (CONFIG_MBEDTLS)
```

Mmm unless I'm missing something this filter might not be correct now: `not CONFIG_MBEDTLS` excludes all scenarios where Mbed TLS is enabled, but that's something we want here. I think we can just leave
```suggestion
        CONFIG_MBEDTLS
```
Wdyt?